WalletWasabi/Wallets/Wallet.cs

```diff
@@ -457,6 +462,17 @@ private async Task LoadWalletStateAsync(CancellationToken cancel)
 				await ProcessFilterModelAsync(filterModel, cancel).ConfigureAwait(false),
 			new Height(bestKeyManagerHeight.Value + 1),
 			cancel).ConfigureAwait(false);
+
+		// Set BestHeight as the last filter tested.
+		var lastFilter = BitcoinStore.IndexStore.GetLastFilter();
```

Comment:
What if you use the `LastProcessedFilter` property?

Reply:
`LastProcessedFilter` is bettern that `GetLastFilter`, yes.